src/server/api/assignment.js

```diff
@@ -113,10 +113,7 @@ export function getContacts(
 export const resolvers = {
   Assignment: {
     ...mapFieldsToModel(["id", "maxContacts"], Assignment),
-    texter: async (assignment, _, { loaders }) =>
-      assignment.texter
-        ? assignment.texter
-        : loaders.user.load(assignment.user_id),
+    texter: async (assignment) => await r.table("user").get(assignment.user_id),
```

Comment:
Yes, please revert this -- we definitely want this data to come from the cache.
The issue you observed IS something we should fix.  If you want to do it in this PR, then it looks like `editUser()` correctly calls ` await cacheableData.user.clearUser(member.id, member.auth0_id);`
The problem is that clearUser doesn't currently clear the loaders (local memory) cache.  What you want to do is copy the pattern found in:
src/server/models/cacheable_queries/organization.js (or campaign.js)
where you import `loaders` from "../../models" and then at the end of clearUser (in user.js in the same dir) you run `loaders.user.clear(Number(id))` and do the same thing with type string (to be extra sure).

Caching is a super-crucial part of scaling Spoke as it's a very 'chatty' client which makes the database the bottleneck, so it's important to retain (and in the future extend) the data that's cached -- especially for anything texter-related.

Reply:
Alright. Will take a look at it.